mobile_app/test/test_checkout_build_repos.py

```diff
@@ -74,6 +90,16 @@ def test_repos_checked_out(self):
             "test-branch-config"
         )
 
+        # And that branch is at the tip
+        self.assertEqual(
+            config_repo.active_branch.commit,
+            config_repo.remotes.origin.refs["test-branch-config"].commit
+        )
+        self.assertEqual(
```

Comment:
Ignorant question....should you assert that you're not on master? (Sorry...didn't look too closely at the mocking.)